bin/react-scripts.js

```diff
@@ -5,14 +5,17 @@ var args = process.argv.slice(3);
 
 switch (script) {
 case 'build':
-case 'start':
 case 'eject':
+case 'start':
```

Comment:
Hmm I'm a bit confused by this.

We only use shell scripts for our own infrastructure (end to end test and release script). We would like all user-ran scripts to be written in JavaScript.

Ideally I'd like to mirror what we do with start and build. Create a JS file in scripts directory called "test.js". It's fine if all it does is call Jest.

Then set that script as "test" command in package.json, identical to how we do build and start. This lets us work on Create React App locally without creating apps all the time. We just run those commands in the root folder, and they work (try npm start).

Finally, npm test needs to be a part of end to end test. To be clear e2e is *our* test that ensures Create React App works. It runs npm start and npm run build in root folder to test our development environment, then creates an app and runs them again to make sure generated app works, then ejects and runs them yet again. So this script is where you want to add "npm test" calls so that we know that testing infrastructure works and won't be broken by future PRs.